contracts/misc/StakedTokenDataProvider.sol

```diff
@@ -1,185 +1,159 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.7.5;
+pragma solidity ^0.7.5;
 pragma experimental ABIEncoderV2;
 
 import {IERC20} from '../interfaces/IERC20.sol';
-import {AggregatedStakedPSYSV3} from '../interfaces/AggregatedStakedPSYSV3.sol';
```

Comment:
To ensure that AggregatedStakedTokenV3 continues to function properly, I left it unmodified and created a new interface
'IAggregatedStakeToken' to integrate it with the 'Interface'. IAggregatedStakeToken includes the methods of StakedPSYS, StakedPSYS2, and StakedTokenV2Rev3, but we only need StakedTokenV2Rev3.

Reply:
Still not clear to me why the change was needed. Did you get this from somewhere or was it your idea? From What I see the change from _AggregatedStakedPSYSV3_ to _AggregatedStakedTokenV3_ removes some GHO's methods that might be useful since we'll release that token together with everything else. 

I just need to understand these changes so maintenance is easier later on.

Reply:
1. By interface you mean the interface repo right? (the frontend). 
2. The code from StakedTokenDataProvider was already on the repo but lacked the removal of BPT and other unused stuff (these changes I understood). What I still don't understand is why you changed the imports.
3. Why do you think there is something missing related to GHO on the interface? (I'm asking bc I haven't looked at that part of the code yet, so this will help me later on :) ) 
4. Is this the code you found on etherscan? https://etherscan.io/address/0x5e045cfb738f01bc73ceaff783f4c16e8b14090b#code

Reply:
1. Yes, but the lending interface in org pegasys-fi is behind by 493 commits compared to aave/interface. They are using the StakingServiceV3, which handles the UI Data of GHO token.
2. I think I can take some time to make fewer changes, including not changing the imports.
3. The code of StakingServiceV3 is in @aave/contract-helpers@v1.28.0.
4. The URL of the code is https://etherscan.io/address/0xb12e82DF057BF16ecFa89D7D089dc7E5C1Dc057B#code.